Apollo/Structures/Offset.cs

```diff
@@ -24,13 +29,27 @@ public class Offset {
                 }
             }
         }
+        
+        double _angle = 0;
+        public double Angle {
```

Comment:
This shouldn't be a part of Offset, but rather a separate list inside of Copy. Offset is reused for Move and it makes no sense to have Angle there, since it's only ever used for interpolating in Copy.